extruct/_extruct.py

```diff
@@ -34,6 +35,9 @@ def extract(htmlstring, base_url=None, encoding="UTF-8",
                  '@type': 'example_type',
                  /* All other the properties in keys here */
                  }
+       return_html_node: if True, it includes into the result a HTML node of
+                         respective embedded metadata under 'htmlNode' key.
```

Comment:
I would add that is only microdata compatible here as well. Some people might just read the comments and miss this piece of information from the docs :)